core/data_access/models/verification_code_model.py

```diff
@@ -1,12 +1,14 @@
-from django.db import models 
+from django.db import models
 
+from core.data_access.models.player.player_model import Player
 
 class VerificationCode(models.Model):
     code_id = models.AutoField(primary_key=True)
+    player = models.OneToOneField(Player, on_delete=models.CASCADE, related_name='verification_code', db_column='playerid')
```

Comment:
Because this feel is non-nullable i have difficulty migrating the updates to the DB -- (i have to set a default value for existing rows.)

Reply:
Have you tried dropping tables and migrating again?